### ISSUE TITLE
Missing react-a11y converter: react-a11y-props

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/tslint-to-eslint-config#900]
### 💥 Missing Converter

-   `tslint-to-eslint-config` version: 2.0.1
-   `typescript-eslint` version: 4.9.1

#### TSLint Rule Name

`react-a11y-props`

Per https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/ROADMAP.md, the equivalent rule is now `jsx-a11y/aria-props`.

#### Package Source

https://github.com/teamairship/tslint-react-a11y
